_posts/2020-05-09-web-scraping.md

```diff
@@ -2,6 +2,12 @@
 layout: post
 author: ["Aashish Kumar", "Lakshya Arora"]
 author_url: ["https://github.com/aashish-ak", "https://github.com/aroralakshya"]
+author_github_id: ["aashish-ak",""]
+author_bio: ["", "Description of Lakshay"]
+author_image_url: ["",""]
+author_twitter_url: ["", "https://twitter.com"]
+author_facebook_url: ["", ""]
+author_instagram_url: ["",""]
```

Comment:
This is the case when there are multiple authors for a single post, like aashish-ak is in authors.yml, so It will add the data from that. 
Now for a moment Lakshay don't have in accounts.yml, so we can have his social link as suggested in this.
If both authors are in authors.yml, then no need of this all only authors_github_id will suffice.
This is the approach I followed to first find in authors.yml, and if not found then the data will be tried to find from authors.yml

Reply:
Great work. We'll review and merge this soon.

Reply:
Ya sure @aashish-ak @aroralakshya